riscv/insns/ssrdp.h

```diff
@@ -0,0 +1,25 @@
+require_extension(EXT_ZICFISS);
+
+reg_t prv = STATE.prv;
+reg_t xSSE = 0;
+
+if (prv == PRV_M)
+  xSSE = 0;
+else if (prv == PRV_S || prv == PRV_HS)
```

Comment:
should we use false or 0 consistently. Does xSSE need to be type reg_t or is a boolean more appropriate?